src/components/organisms/EarthEditor/OutlinePane/hooks.tsx

```diff
@@ -245,7 +253,8 @@ export default () => {
     sceneDescription,
     selectedType: selected?.type,
     selectedLayerId: selected?.type === "layer" ? selected.layerId : undefined,
-    selectedWidgetId: selected?.type === "widget" ? selected.widgetId : undefined,
+    selectedWidgetId:
+      selected?.type === "widget" ? `${selected.pluginId}/${selected.extensionId}` : undefined,
```

Comment:
I know we aren't supporting multi-widgets yet, but is there a reason why you are using `pluginId/extensionId` instead of just `widgetId` OR `widgetId/pluginId/extensionId`?
I could see we use this `selectedWidgetId` inside the **TreeView** so if we are using this value on updating `selectedWidget` shouldn't we be using all three?

Reply:
Widgets (note: extensions, not scene widgets) are always displayed on the left panel even when there are no widgets on the scene so no widget ID exists. For now multiple widget is supported only by back-end, not front-end. To identify widgets in that case, we have to treat a set of pluginId and extensionId.

Reply:
Gotcha!